operator/deploy/role.yaml

```
@@ -32,6 +32,23 @@ rules:
   - get
   - list
   - patch
+- apiGroups: [""]
+  resources:
+  - secrets
+  verbs:
+  - create
```

let's do this now while we have it in a PR... add "list" and "watch" permissions here.
```suggestion
  - create
  - list
  - watch
```